chess_engine.py

```diff
@@ -174,6 +192,7 @@ def get_valid_moves(self, starting_square):
                     if can_move:
                         valid_moves.append(move)
                 self._is_check = True
+                logger.debug(f"{'White' if self.white_turn else 'Black'} is chess position")
```

Comment:
Be careful, your code doesn't run as expected - Your game stops because of logger problems.